ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -133,392 +139,351 @@ Control {
 
-                        Control {
-                            width: 24
-                            height: 24
-                            anchors.left: parent.left
-                            anchors.leftMargin: 12
-                            anchors.bottom: parent.bottom
-                            anchors.bottomMargin: 12
-                            visible: !!model.communityImage
-
-                            background: Rectangle {
-                                radius: parent.width/2
-                                color: Theme.palette.indirectColor2
-                            }
-                            contentItem: StatusRoundedImage {
-                                anchors.fill: parent
-                                anchors.margins: 4
-                                image.fillMode: Image.PreserveAspectFit
-                                image.source: model.communityImage
-                            }
+                        contentItem: StatusBaseText {
+                            id: amountText
+                            horizontalAlignment: Text.AlignHCenter
+                            verticalAlignment: Text.AlignVCenter
+                            font.pixelSize: Style.current.asideTextFontSize
+                            text: "x"+model.userHas
                         }
-                        HoverHandler {
-                            id: hhandler
-                            cursorShape: hovered ? Qt.PointingHandCursor : undefined
+                    }
+
+                    StatusFlatRoundButton {
+                        implicitWidth: 24
+                        implicitHeight: 24
+                        anchors.right: parent.right
+                        anchors.rightMargin: 12
+                        anchors.top: parent.top
+                        anchors.topMargin: 12
+                        visible: (hoverHandler.hovered && model.communityId === "")
+                        type: StatusFlatRoundButton.Type.Secondary
+                        icon.name: "external"
+                        icon.width: 16
+                        icon.height: 16
+                        radius: width/2
+                        icon.color: Theme.palette.directColor1
+                        onClicked: {
+                            Global.openLinkWithConfirmation(model.permalink, model.domain)
                         }
+                    }
 
-                        TapHandler {
-                            onSingleTapped: {
-                                //TODO https://github.com/status-im/status-desktop/issues/13702
-                                Global.openLink(model.permalink)
+                    ExpandableTag {
+                        id: communityTag
+                        visible: !!model.communityImage
+                        tagHeaderText: model.name
+                        tagName: model.communityName
+                        tagImage: model.communityImage
```

Comment:
We don't have this roleName provided in the collectibles model.
These are the roleNames we currently have:
```
      CollectibleRole.Uid.int:"uid",
      CollectibleRole.ChainId.int:"chainId",
      CollectibleRole.ContractAddress.int:"contractAddress",
      CollectibleRole.TokenId.int:"tokenId",
      CollectibleRole.Name.int:"name",
      CollectibleRole.MediaUrl.int:"mediaUrl",
      CollectibleRole.MediaType.int:"mediaType",
      CollectibleRole.ImageUrl.int:"imageUrl",
      CollectibleRole.BackgroundColor.int:"backgroundColor",
      CollectibleRole.CollectionUid.int:"collectionUid",
      CollectibleRole.CollectionName.int:"collectionName",
      CollectibleRole.CollectionSlug.int:"collectionSlug",
      CollectibleRole.IsLoading.int:"isLoading",
      CollectibleRole.Ownership.int:"ownership",
      CollectibleRole.CommunityId.int:"communityId",
      CollectibleRole.CommunityName.int:"communityName",
      CollectibleRole.CommunityColor.int:"communityColor",
      CollectibleRole.CommunityPrivilegesLevel.int:"communityPrivilegesLevel",
      CollectibleRole.TokenType.int:"tokenType",
```

@dlipicar Would it be a good idea to add the community image as well in the collectibles model?

Reply:
Created a task to do this. I don't see any other way. I was thinking we could do a left join with the global communities model, but we might not have the community available there 🤔

Reply:
I see we have the community image here https://github.com/status-im/status-desktop/blob/master/src/backend/collectibles_types.nim#L295. Could be just a matter of exposing it in the model

Reply:
@alexjba yeap, the community imageUrl is already there. It's even exposed in `src/app/modules/shared_models/collectibles_entry.nim`, just missing in the model

Reply:
@alexandraB99 permalink doesn't currently come from the backend, we build it using the collectible ID. See this for reference: https://github.com/status-im/status-desktop/pull/14170/files#diff-49a543b84eef8685a0154567966dc0bfb8238b733a8beeb4149988a75604f4b8R587

Not sure what the `domain` is though

Reply:
@dlipicar I see there you use `collectible.networkShortName, collectible.contractAddress, collectible.tokenId` but in the model roles above, at least, I do not see `networkShortName` neither `contractAddress`.